client/src/pages/CreateSurvey/index.jsx

```diff
@@ -1,7 +1,81 @@
-import React from "react";
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { TextField, Button, Switch } from '@material-ui/core';
 
 const CreateSurvey = () => {
-    return <></>;
+  const dispatch = useDispatch();
+  const setMetadata = (event, inputType) => {
+    const payload = {};
+    payload[event.target.name] =
+      inputType === 'switch' ? event.target.checked : event.target.value;
+    dispatch({ type: 'SET_METADATA', payload });
+  };
+  const {
+    title,
+    description,
+    recipients,
+    disclaimer,
+    anonymous,
+    // eslint-disable-next-line
+    questions,
+  } = useSelector((state) => state.createSurveyReducer);
+  const handleSubmit = () => {};
+  return (
+    <form onSubmit={handleSubmit}>
+      <TextField
+        required
+        error={title.length < 5 || title.length > 100}
+        helperText={
+          title < 5 || title > 100
+            ? 'Title must be between 5 and 100 characters!'
+            : ''
+        }
```

Comment:
The turnary operation should be decoupled from the return statement and should be a descriptive const variable.

Reply:
This operator is only used once so it seems excessive to be allocating memory with a const when the code above is perfectly readable. This is @marthaklambert's code though so it's up to her.